hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -657,8 +657,9 @@ private FileStatusAdapter getFileStatusForKeyOrSnapshot(
       OFSPath ofsPath, URI uri, Path qualifiedPath, String userName)
       throws IOException {
     String key = ofsPath.getKeyName();
+    OzoneBucket bucket = null;
```

Comment:
Nit: unnecessary change.  Variable `bucket` only used in scope of try block and can remain declared there.